src/cli/commands/lint.ts

```diff
@@ -91,7 +91,7 @@ linting ./openapi.yaml
 
     if (ruleset) {
       rules = await tryReadOrLog(this, async () => {
-        return readRulesFromRulesets(...ruleset);
+        return Promise.all(ruleset.map(async file => readRulesFromRuleset((await readParsable(file, 'utf-8')).data)));
```

Comment:
Can we break this line down a bit? It’s doing a lot of things and readability suffers.

Reply:
Agree, and perhaps add a little comment with brief explanation of what this is doing.